.dockerignore

```diff
@@ -0,0 +1,3 @@
+.git
+.env
+.dockerignore
```

Comment:
`.dockerignore` lists out files we don't want copied from the local filesystem to the dockerized app when building the containers. See [the docs](https://docs.docker.com/engine/reference/builder/#/dockerignore-file) for more info